test/demo/pet-store-env.sh.yml

```diff
@@ -33,84 +33,84 @@ spec:
       labels:
```

Comment:
do we want to remove the demo altogether? i am guessing that the dev env replaces that

Reply:
yeah demo kinda totally does not work

Reply:
removed